frontend/src/static/js/v_authorship.js

```diff
@@ -40,14 +40,13 @@ window.vAuthorship = {
       totalBlankLineCount: '',
       filesSortType: 'lineOfCode',
       toReverseSortFiles: true,
-      hasActiveFile: true,
       filterSearch: '*',
     };
   },
 
   watch: {
     selectedFiles() {
-      setTimeout(this.updateCount, 0);
+      this.displayMoreFiles();
```

Comment:
unresolved function

Reply:
removed

Reply:
thanks for re-reviewing this as well!

I suppose the changes mentioned above are fine on their own?
I've updated the commit message, title and linked issues to reflect, let me know otherwise!

Reply:
yes I feel the current implementation are fine